docs/utils/nft-borsh.ts

```diff
@@ -138,18 +141,53 @@ export const NFTOKEN_COLLECTION_ACCOUNT = new FixableGlowBorsh<{
   ],
 });
 
-export const SYSTEM_CREATE_ACCOUNT_IX = new FixableGlowBorsh<{
-  // TODO: Avoid passing it, creating a dedicated codec instead.
-  instruction: number; // Must be `1`.
-  lamports: Solana.SolAmount;
-  // FIXME: Do we have a better type for `u64`?
-  space: Solana.SolAmount;
-  program_id: Solana.Address;
+// Schema: https://docs.rs/solana-program/1.10.6/solana_program/system_instruction/enum.SystemInstruction.html#variant.CreateAccount
+// Example transaction: https://solscan.io/tx/3WjJYXTaaeyn5j5EDzYPPU9pQuPM7rJmXxaLNV1LWxHA5YgAfTi1XfEaX8bQGRJqfUiogLPJqFVer5QCxynHkdCY
+// - 00000000 - discriminator
+// - c0e1e40000000000 - amount (15_000_000 lamports)
+// - 5200000000000000 - space (82)
+// - 06ddf6e1d765a193d9cbe146ceeb79ac1cb485ed5f5b37913a8cf5857eff00a9 - program owner (TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA)
+
+/**
+ * This is a four bytes discriminator, used in system program.
+ */
+const discriminatorU32 = (hex: string): FixedSizeBeet<null, null> => {
+  if (hex.length !== 8) {
+    throw new Error("u32 discriminator must be exactly 8 characters long.");
+  }
+  return {
+    write: function (buf: Buffer, offset: number) {
+      const stringBuf = Buffer.from(hex, "hex");
+      assert.equal(stringBuf.byteLength, 4, `${hex} has invalid byte size`);
+      stringBuf.copy(buf, offset, 0, 4);
+    },
+
+    read: function (buf: Buffer, offset: number): null {
+      const stringSlice: Buffer = buf.slice(offset, offset + 4);
+      const expected = stringSlice.toString("hex");
+      if (expected !== hex) {
+        throw new Error("Did not get expected value.");
+      }
+
+      return null;
+    },
+
+    length: 4,
+    byteSize: 4,
+    description: `DiscriminatorU32`,
+  };
+};
+
+export const SYSTEM_CREATE_ACCOUNT_IX = new GlowBorsh<{
+  ix_discriminator: null;
+  amount: Solana.SolAmount;
+  space: BN;
+  program_owner: Solana.Address;
 }>({
   fields: [
-    ["instruction", u32],
-    ["lamports", GlowBorsh.solAmount],
-    ["space", GlowBorsh.solAmount],
-    ["program_id", GlowBorsh.address],
+    ["ix_discriminator", discriminatorU32("00000000")],
+    ["amount", GlowBorsh.solAmount],
```

Comment:
Do I understand correctly that the names of the fields don't matter, only the order?